lib/entities/organization-membership.js

```diff
@@ -0,0 +1,128 @@
+ * @prop {string} role
+ * @prop {boolean} status - User is an admin
+ * @prop {function(): Object} toPlainObject() - Returns this Organization Membership as a plain JS object
+ */
+
+/**
+ * @memberof OrganizationMembership
+ * @typedef OrganizationMembershipCollection
+ * @prop {number} total
+ * @prop {number} limit
+ * @prop {number} skip
+ * @prop {Object<{type: "Array"}>} sys
+ * @prop {Array<OrganizationMembership.OrganizationMembership>} items
+ * @prop {function(): Object} toPlainObject() - Returns the collection as a plain JS object
+ */
+
+function createOrganizationMembershipApi (http, orgId) {
+  const baseURL = http.defaults.baseURL.replace('/spaces/', `/organizations/${orgId}`)
+
+  return {
+
+    /**
+     * Sends an update to the server with any changes made to the object's properties
+     * @memberof OrganizationMembership
+     * @func update
+     * @return {Promise<OrganizationMembership>} Object returned from the server with updated changes.
+     * @example
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * client.getOrganizations()
+     * .then((organizations) => organizations.items[0])
+     * .then(org => org.getOrganizationMembership('organizationMembership_id'))
+     * .then((organizationMembership) => {
+     *  organizationMembership.role = 'member';
+     *  organizationMembership.update();
+     * })
+     * .catch(console.error)
+     */
+
+    update: function () {
+      const raw = this.toPlainObject()
+      const data = cloneDeep(raw)
+      delete data.sys
+      delete data.status
+      delete data.user
+      return http.put('organization_memberships' + '/' + this.sys.id, data, {
+        headers: {
+          'X-Contentful-Version': this.sys.version || 0
+        },
+        baseURL
+      })
+        .then((response) => wrapOrganizationMembership(http, response.data), errorHandler)
+    },
+
+    /**
+     * Deletes this object on the server.
+     * @memberof OrganizationMembership
+     * @func delete
+     * @return {Promise} Promise for the deletion. It contains no data, but the Promise error case should be handled.
+     * @example
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * client.getOrganizations()
+     * .then((organizations) => organizations.items[0])
+     * .then(org => org.getOrganizationMembership('organizationMembership_id'))
+     * .then((organizationMembership) => {
+     *  organizationMembership.delete();
+     * })
+     * .catch(console.error)
+     */
+
+    delete: function () {
+      return http.delete('organization_memberships' + '/' + this.sys.id, {
+        headers: {
+          'x-contentful-enable-alpha-feature': 'organization-user-management-api'
```

Comment:
delete needs the alpha header but update doesnt? 🤔

Reply:
@phoebeschmidt yeah, as far as I can see in the stoplight...

Reply:
alright! sure. resolved :)

Reply:
@phoebeschmidt I will check it with BE. maybe it's a bug

Reply:
@phoebeschmidt actually, BE just started to work on the removal of the alpha header for this endpoint, so I would need to adjust the implementation soon.